buildspecs/integ-test.yml

```diff
@@ -0,0 +1,6 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+    - mvn clean verify -Dskip.unit.tests -P integration-tests -Dfindbugs.skip -Dcheckstyle.skip -pl !:dynamodbmapper-v1 -Dfailsafe.rerunFailingTestsCount=1
```

Comment:
I think this might require the `-Dmaven.wagon.httpconnectionManager.maxPerRoute=2` option too, since this command will be pulling down dependencies from Maven as well

Reply:
It's currently working without it, but yeah, I can add it